src/App.js

```diff
@@ -1,19 +1,53 @@
+import { render } from '@testing-library/react';
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import {useState} from 'react'; 
+
+
+
+const INITIAL_ENTRIES = chatMessages
 
 const App = () => {
+  const initialCopy = INITIAL_ENTRIES.map(chatEntry => {
+    return {...chatEntry};
+  })
+
+  
+
+  const [chatLog] = useState(initialCopy);
```

Comment:
This state does not ever get updated and does not need to be set up like this. Since `chatLog` is never updated, it works to just use `initialCopy`.